src/Giraffe/ResponseWriters.fs

```diff
@@ -9,10 +9,13 @@ open Microsoft.AspNetCore.Http
 open Microsoft.Net.Http.Headers
 open FSharp.Control.Tasks.V2.ContextInsensitive
 open Giraffe.GiraffeViewEngine
+open Microsoft.IO
+
 
 // ---------------------------
 // Helper functions
 // ---------------------------
+let recyclableMemoryStreamManager = RecyclableMemoryStreamManager()
```

Comment:
I possibly could move this to either a Singleton with IoC or some other location since we are using it twice now.